CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## PENDING
+
+### Features
+- Adding the ability to use refresh tokens with single page applications (SPA) (Early Access feature - reach out to our support team)
+  - `scopes` configuration option now handles 'offline_access' as an option, which will use refresh tokens IF your client app is configured to do so in the Okta settings
+    - If you already have tokens (from a separate instance of auth-js or the okta-signin-widget) those tokens must already include a refresh token and have the 'offline_access' scope
+    - 'offline_access' is not requested by default.  Anyone using the default `scopes` and wishing to add 'offline_access' should pass `scopes: ['openid', 'email', 'offline_access']` to their constructor (FIXME: Is constructor enough?)
```

Comment:
Aaron has a fix for 4.1, should be backported after merging https://github.com/okta/okta-auth-js/pull/535